### PR TITLE
feat(): verify party proxy contract on etherscan for new parties

### PR DESCRIPTION
**Pull request by marcusmolchany**
on `StartParty` events we verify the party's proxy contract on etherscan

etherscan docs: https://docs.etherscan.io/api-endpoints/contracts#verifying-proxy-contract-using-curl

### tested locally
```sh
➜  party-feed git:(marcus/feat/verify-party-proxy) ✗ yarn dev
yarn run v1.22.17
$ tsc && node ./transpiled/index.js
verifying proxy for contract 0x615426d6296a96a3fed86a37f3372e4eddc76c40
data {
  status: '1',
  message: 'OK',
  result: 'egjzwkkne63wa8hj2fl9r3ahp22b23eawcavmtim5us3jvnpdc'
}
Successfully verified proxy contract for 0x615426d6296a96a3fed86a37f3372e4eddc76c40. guid egjzwkkne63wa8hj2fl9r3ahp22b23eawcavmtim5us3jvnpdc.
verified
✨  Done in 2.63s.
```

### todo
- [x] add etherscan api key to party-feed render.com

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ export const config = {
     process.env.DISCORD_WEBHOOK_URL,
     "discord webhook url"
   ),
+  etherscan: {
+    apiBase: "https://api.etherscan.io/api",
+    apiKey: stringFromENVorThrow(
+      process.env.ETHERSCAN_API_KEY,
+      "twitter api key"
```
Comment (steveklebanoff) [inline src/config.ts]: etherscan

File: src/etherscan.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    return false;
+  }
+};
+
+/**
+ * verify proxy contract on etherscan if the event is a start event
+ * @param {PartyEvent} event
+ * @returns true/false if verified
+ */
+export const verifyProxyContractForStartEvent = async (
```
Comment (steveklebanoff) [inline src/etherscan.ts:50]: is it worth it to have such a tiny fn? perhaps just combine this with `verifyProxyContract`

**Review by steveklebanoff — APPROVED** [review]
two nits but LGTM

File: src/etherscan.ts (hunk 2 above)
Comment (marcusmolchany) [reply src/etherscan.ts:50]: i split them up so that i could call the function w/ an address when i was testing locally

**Pull request merged**